File: 백준/Bronze/2420. 사파리월드/사파리월드.cs
using System;

class Program
{
    static void Main()
    {
        string[] inputs = Console.ReadLine().Split(' ');
        long N = long.Parse(inputs[0]);
        long M = long.Parse(inputs[1]);

        if (N < M)
        {
            Console.WriteLine(M - N);
        }
        else
        {
            Console.WriteLine(N - M);
        }
    }
}

File: 백준/Bronze/2420. 사파리월드/사파리월드.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question('', (input) => {
    const [N, M] = input.split(' ').map(Number);

    if (N < M) {
        console.log(M - N);
    } else {
        console.log(N - M);
    }

    rl.close();
});